### PR TITLE
Resources page

### PR DESCRIPTION
**Pull request by Christian-MK**
Creation of an open source disclosure page draft

### CODE REVIEW

**Comment by ross-spencer** [comment]
Thanks Christian. This looks like a decent chunk of work. 

I have two initial comments. Both largely around sustainability of approach.

## Editorially

I feel as an organization we keep mixing marketing with information. So we have a page here for disclosures but we mix it with our ethos. Maybe a sentence works, but we're repeating more from our various whitepapers and so on, e.g. more on _trust but verify_; describing our own approach as 'ambitious'.

I also think we're also rationalizing things that do not need rationalizing, e.g. describing the point of the page, i.e. in order to ... meet responsibilities ... to become a dao ...; benefits of open source etc..

I can't foresee a time we would ever need the content of this disclosure page, without also repeating this context elsewhere, e.g. in a whitepaper, or more general section higher up in the docs.

Possible editorial solutions:
* are we missing a mission statement and values statement somewhere in the docs?
* is there a page higher up already where our open source ethos can be described? (maybe it just needs to be separate from the disclosure page)

## Technically

This might be fine prior to v1 but I feel like there's a bigger discussion about sustainability and creating some of our outputs (the balance between automation and manual output).

Either way, the approach of manually writing these docs does look like a lot of work and fine as a one off, but I feel we'll struggle to keep this up to date with new information.

The question for me is what is the primary entry point for anyone who wants to see the open source face of Orcfax? -- Github. So how we we integrate that as simply as possible without redundancy?

One low-tech approach might be to just link out to it:

* Our public repos as the public sees them: https://github.com/orcfax/?q=&type=public&language=&sort=

A more hi-tech approach might be to consider how we build this page dynamically (or host it on a separate disclosures repo, e.g. like the glossary build). Here is a curl command that can be used to generate a simplified list of all our non-forked public repositories and their license information:

```curl
curl https://api.github.com/users/orcfax/repos \
 | jq '.[] 
 | select(.fork != true) 
 | { name: .name, 
     url: .html_url, 
     description: .description, 
     license: .license.name, 
     "license url": .license.url 
   }'
```

And yet to be investigated, do we have any docusaurus plugins available to us that might support the above curl? or some other github integration?

Or even just a shell script associated with this repo that can output some automated html snippets might be useful?

obviously we can be here to support anything needed.

----

In summary are we able to rework some of this so the disclosures is a basic table, even manually generated for now? And move some of the bigger picture stuff to some bigger picture pages in elsewhere?

Also, is it worth trying to put a docs meeting together when Peter is back online to discuss further? (I think that discussion might include getting a mission statement and values page up (I feel like these are nearly all there across the various pieces we have)). I'd also like to discuss maintenance/sustainability and figure out if there's a more practical way we can reduce complexity/redundancy.

We still need to follow up on https://diataxis.fr/ and I wonder if we should create an issue for this and start to rework the docs? (i believe you asked the same recently in discord)

**Comment by Christian-MK** [comment]
> Editorial

Point well taken! I have cut out all the marketing language and moved it to the-orcfax-vision#open-source-vs-closed-source

**Comment by Christian-MK** [comment]
> The question for me is what is the primary entry point for anyone who wants to see the open source face of Orcfax? -- Github. So how we we integrate that as simply as possible without redundancy?

Agree 100%
I had a comment inline saying as much. I'm all for the conversation and implementing whatever strikes the desired balance

**Comment by Christian-MK** [comment]
Given the feedback from Ross, closing in order to pursue more effective means of executing
